docs/proposals/metrics-api.md

```diff
@@ -0,0 +1,134 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
There is still a link to the previous proposal. The problem with the previous proposal is that its first version is a mix of Master Metrics API and Kubelet Metrics API which is confusing and the second version was not merged.